assets/src/design-system/components/pill/pill.js

```diff
@@ -60,13 +61,16 @@ const StyledPill = styled.button(
   `
 );
 
-function Pill({ children, isActive, onClick, ...rest }) {
+const Pill = forwardRef(function Pill(
```

Comment:
Won't this cause warnings about using prop types with `forwardRef`?

What we usually do is something like this:

```js
function Pill({ children, isActive, onClick, ...rest }, ref)  { ... }

const PillWithRef = forwardRef(Pill);

Pill.propTypes = {...}

export default PillWithRef;
```

Reply:
@swissspidy 😂 I actually thought it was the other way around!

I switched em around in storybook and the editor, neither one seems to cause warnings or errors for me, although I swore I saw them with the components that were implemented with the `forwardRef` **not** inline with the component definition. I can't reproduce them now, though!

Reply:
That said, I believe these changes come from a different PR: https://github.com/google/web-stories-wp/pull/6977

Reply:
AH! Here! I found an example (sorry Max for multithreading on your PR) :D 
```
Warning: forwardRef render functions do not support propTypes or defaultProps. Did you accidentally pass a React component? react.development.js:315
    React 3
    js textSet.js:156
    Webpack 25
```
![Screen Shot 2021-03-31 at 5 19 24 PM](https://user-images.githubusercontent.com/41136059/113226532-4b615200-9245-11eb-9dd6-5b128ff80fef.png)


Looking at either of those it's got lines like these:

`export default forwardRef(PageLayout);`
`export default forwardRef(TextSet);`

And for those two components I got the error to go away by wrapping it inline with `forwardRef` 🤷 Kinda weird that it only shows on those components?

Reply:
Coincidentally, those two are actually two places I updated yesterday to the method I mentioned above. Here's the `PageLayout` one on `main`:

https://github.com/google/web-stories-wp/blob/3835918935ac7ced536f6772fc63fd765173f856/assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js#L130-L142


But perhaps the version in this PR works too? 🤷